docs/tutorials/intro.md

```diff
@@ -166,7 +167,7 @@ fn main() {
         ..Default::default()
     }.into();
 
-    let dtf = DateTimeFormat::try_new(locale!("ja"), &provider, &options)
+    let dtf = DateTimeFormat::try_new_with_buffer_provider(&provider, &locale!("ja").into(), &options)
         .expect("Failed to initialize DateTimeFormat");
 
```

Comment:
Issue: Re-generate `work_log.wasm` with the smaller provider. Or, just don't re-generate it in this commit since the data keys didn't change, and that's all it is used for.